recipes/ntv2/all/conanfile.py

```diff
@@ -0,0 +1,122 @@
+import os
+
+from conan import ConanFile
+from conan.tools.cmake import CMake
+from conans import tools
+
+required_conan_version = ">=1.43.0"
+
+
+class Ntv2Conan(ConanFile):
+    name = "ntv2"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/aja-video/ntv2"
+    description = "AJA NTV2 SDK"
+    topics = "video, hardware"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    generators = "CMakeToolchain"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+                  destination=self._source_subfolder, strip_root=True)
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure(build_script_folder=self._source_subfolder)
+        cmake.build()
+
+    def package(self):
+        self.copy(os.path.join(self._source_subfolder, 'LICENSE'), 'licenses')
+
+        for suffix in ["so", "lib", "a", "dylib", "bc"]:
+            self.copy(
+                f"*.{suffix}", src=os.path.join("ajalibraries", "ajantv2"), dst="lib", keep_path=False)
+        if self.settings.os == "Windows" and self.options.shared:
+            self.copy(
+                "*.dll", src=os.path.join("ajalibraries", "ajantv2"), dst="bin", keep_path=False)
+        for lib in ["ajaanc", "ajacc", "ajantv2"]:
+            self.copy("*", src=os.path.join(self._source_subfolder,
+                      "ajalibraries", lib, "includes"), dst=os.path.join("include", lib))
+        self.copy("ajalibraries/**/*.h",
+                  src=self._source_subfolder, dst="include")
+        self.copy("ajalibraries/**/*.hh",
+                  src=self._source_subfolder, dst="include")
```

Comment:
there is an install target, why not relying on it?

Reply:
it was added in 16.2